### PR TITLE
Update win_susp_calc.yml

### PR DESCRIPTION
**Pull request by qwertysn35**
Changed the condition.

### CODE REVIEW

File: rules/windows/process_creation/win_susp_calc.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ detection:
         Image|endswith: '\calc.exe'
     filter2:
         Image|contains: '\Windows\Sys'
```
Comment (frack113) [inline rules/windows/process_creation/win_susp_calc.yml:21]: as path can be "C:\Windows\SysWOW64" or "C:\Windows\System32" can change  to be more accurate
`Image|contains: ':\Windows\Sys'`

File: rules/windows/process_creation/win_susp_calc.yml [hunk 2]
```diff
@@ -19,7 +19,7 @@ detection:
         Image|endswith: '\calc.exe'
```
Comment (frack113) [inline rules/windows/process_creation/win_susp_calc.yml:19]: more compact: 
```yaml
    select_calc:
       - CommandLine|contains: '\calc.exe '
       - Image|endswith: '\calc.exe'
    filter2:
        Image|contains: ':\Windows\Sys'
    condition: select_calc and not filter2
```

**Comment by Neo23x0** [comment]
No, the original condition was correct. 

`selection1` looks for a command line in which the command line running calc.exe contains a " " (space) after the calc.exe.

**Comment by qwertysn35** [comment]
Hi @Neo23x0,
Thanks for the feedback. I did not catch the "space" character.

**Pull request closed without merging**